src/fhirtypes/ElementDefinition.ts

```diff
@@ -1326,6 +1327,9 @@ export class ElementDefinition {
       const type = this.type[0].profile?.[0] ?? this.type[0].code;
       const def = resolve(type);
       if (def) {
+        if (!def.complete) {
+          logger.debug(`${def.name} is incomplete and may be missing required fields.`);
```

Comment:
Will any existing test cases execute this logging statement? If not, could you add a test case that does?

Reply:
Good catch! 
I don't think any existing test cases will log this. I will add this test case if still relevant after the work that @cmoesel is working on is complete.